app/helpers/self_service/breadcrumbs_helper.rb

```diff
@@ -62,5 +66,9 @@ def crumb_firm_adviser_new
     def crumb_trading_name_new
       { name: t('self_service.trading_name_new.breadcrumb', firm_name: @firm.registered_name) }
     end
+
+    def crumb_principal_edit
+      { name: t('self_service.principal_edit.breadcrumb', firm_name: @firm.registered_name) }
+    end
   end
 end
```

Comment:
It seems like there's quite a lot going on in this helper but there are no unit tests other than the ones in `application_helper_spec.rb`, which seem quite high level. Is this being covered elsewhere?